README.md

```diff
@@ -33,6 +33,37 @@ See our [Trello](https://trello.com/b/jItetqYe/open-brush) board for more inform
 * [SiMonk0](http://www.furjandesign.com/) for the great new logo!
 * The [SideQuest](https://sidequestvr.com/) team for your support.
 
+## Development / Contributing
```

Comment:
Should we move this to CONTRIBUTING.md and update it from the template originally provided? There may be a larger overall discussion about what belongs in README and what belongs in CONTRIBUTING

Reply:
Sure; do you want me to do this now or was this more of a note for the future?

Reply:
Done (I left the original Contributing there as well for now)